asdf_astropy/converters/fits/fitswcs.py

```diff
@@ -0,0 +1,19 @@
+from asdf.extension import Converter
+
+
+class FitsWCSConverter(Converter):
+    tags = ("tag:astropy.org:astropy/fits/fitswcs-*",)
+    types = ("astropy.wcs.wcs.WCS",)
+
+    def from_yaml_tree(self, node, tag, ctx):
+        from astropy.wcs import WCS
+
+        return WCS(node["hdu"][0].header, fobj=node["hdu"])
+
+    def to_yaml_tree(self, wcs, tag, ctx):
+        node = {}
+        if wcs.sip is not None:
+            node["hdu"] = wcs.to_fits(relax=True)
```

Comment:
```suggestion
            node["hdu"] = wcs.to_fits(relax=True)
```
Presumably the relax is here for the "non-standard" wcs keywords. Will these only be present if `sip` is not None?

Reply:
Is there a case we are not considering? If so, could you please explain how we can account for it?